mdata/cache/ccache.go

```diff
@@ -232,12 +262,8 @@ func (c *CCache) Search(ctx context.Context, metric schema.AMKey, from, until ui
 
 		accnt.CacheChunkHit.Add(len(res.Start) + len(res.End))
 		go func() {
-			for _, hit := range res.Start {
-				c.accnt.HitChunk(metric, hit.Ts)
-			}
-			for _, hit := range res.End {
-				c.accnt.HitChunk(metric, hit.Ts)
-			}
+			c.accnt.HitChunks(metric, res.Start)
+			c.accnt.HitChunks(metric, res.End)
```

Comment:
i think in the majority of cases `len(res.End) == 0`, so we could save a few function calles by only doing this `if len(res.End) > 0`

Reply:
seems cleaner to put that optimization in the HitChunks implementation. will do that